pkg/kubelet/apis/config/validation/validation.go

```diff
@@ -55,8 +56,8 @@ func ValidateKubeletConfiguration(kc *kubeletconfig.KubeletConfiguration, featur
 	if kc.NodeLeaseDurationSeconds <= 0 {
 		allErrors = append(allErrors, fmt.Errorf("invalid configuration: nodeLeaseDurationSeconds must be greater than 0"))
 	}
```

Comment:
Would it make sense to move validation of `EnforceNodeAllocatable` (lines 208-L227 below) to this place?
If this is possible then below check can be done in a simpler way.

BTW, `NodeConfig.NodeAllocatableConfig.EnforceNodeAllocatable` [is a set](https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/cm/container_manager.go#L168). Is there any validation code to check that these options don't get repeated?

Regardless of the above, would the code benefit of changing type of the `EnforceNodeAllocatable` to set?

Reply:
Thanks for your review @bart0sh 
Changing the type of the `EnforceNodeAllocatable` to set will affect the `k8s.io/kubelet/config/v1beta1` API design. That can lead to big code changes. 
I reorganized the code as you suggested and added the duplicate checks. PTAL.

Reply:
/lgtm